example_packages/metapackage_mpi/fpm.toml

```diff
@@ -0,0 +1,15 @@
+name = "test_mpi"
```

Comment:
Can we keep to the manifest simple, something like

```toml
name = "test_mpi"
dependencies.mpi = true
```

would already be sufficient.

Reply:
yeah, output from `fpm new` is a bit verbose

Reply:
Thank you, addressed in commit [76b34ea](https://github.com/fortran-lang/fpm/pull/859/commits/76b34ea1f355464375219113b59c03a73fab0f7f)